include/mesh/mesh_tetgen_interface.h

```diff
@@ -155,6 +159,12 @@ class TetGenMeshInterface
   std::vector<unsigned> _sequential_to_libmesh_node_map;
 
   /**
+   * Parameter controling the behaviour of tetgen.
+   * By default quiet.
+   */
+  std::string _switches = "Q";
```

Comment:
I believe this (inline initialization of non-const class members) is a C++11 only feature, although it was not caught by our "no C++11" test for some reason.